source/install/mattermost-omnibus.rst

```diff
@@ -0,0 +1,117 @@
+Installing Mattermost Omnibus
+============================
+
+Overview
+---------
+
+Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
+
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. 
+
+Prerequisites
+-------------
+
+- A clean Ubuntu server (18.04. Or 20.04)
+- Root level access to the server
+
+.. note:: 
+
+  With Mattermost Omnibus, the ``config.json`` file no longer exists. So you’ll need to use mmctl to make changes to your Mattermost server configuration using ``mmctl config edit``. If you wish to manage your server locally, you may want to enable local mode for mmctl.
+
+Installing Mattermost Omnibus
+------------------------------
+
+During installation you can provide an existing domain name that points to the Ubuntu server or, if you don't have one available, you can log in using the server's IP address.
+
+.. note::
+  Plugin uploads and HTTPS are enabled by default. These settings are modified in the ``yaml`` file using `mmctl config edit <https://docs.mattermost.com/administration/mmctl-cli-tool.html#mmctl-config-edit>`__. 
+
+The Omnibus repositories are configured using a cURL command: 
+
+``curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+This command installs and configures a ``PostgreSQL`` database, an ``nginx`` web server to act as a proxy, and ``certbot`` to issue and renew the SSL certificate. 
```

Comment:
This command configures the repositories needed for those three components rather than installing the components themselves